crates/rover-client/src/operations/subgraph/check_workflow/runner.rs

```diff
@@ -387,6 +410,41 @@ fn get_proposals_response_from_result(
     }
 }
 
+fn get_custom_response_from_result(
```

Comment:
It doesn't feel great that we're duplicating the code just because it has to work for `graphs` and `subgraphs` could we make `get_custom_response_from_result` take either kind of result then this could live in a `utils` package or something similar?

Reply:
I agree, but I'm struggling to think of a way to implement. These functions basically take the response which uses autogenerated types from their respective Queries, which is different between `graph` and `subgraph` and maps them to the common type under `shared`